src/Reducer.test.js

```diff
@@ -15,18 +15,30 @@ import {
 
```

Comment:
This test file should be split out in the same way you've split out your reducers. Each reducer ~can be~ should be tested independently.

Reply:
we'll do it in another branch